updater/spec/dependabot/dependency_snapshot_spec.rb

```diff
@@ -140,59 +140,6 @@
     end
   end
```

Comment:
please is there a way to have unit test coverage for some of the added methods too?